lib/cartopy/crs.py

```diff
@@ -1531,6 +1531,85 @@ def y_limits(self):
         return self._ylim
 
 
+class AlbersEqualArea(Projection):
+    """
+    An Albers Equal Area projection
+
+    This projection is conic and equal-area, and is commonly used for maps of
+    the conterminous United States.
+    """
+
+    def __init__(self, central_longitude=0.0, central_latitude=0.0,
+                 false_easting=0.0, false_northing=0.0,
+                 standard_parallels=(20.0, 50.0), globe=None):
+        """
+        Kwargs:
+
+            * central_longitude - The central longitude. Defaults to 0.
+            * central_latitude - The central latitude. Defaults to 0.
+            * false_easting - X offset from planar origin in metres.
+                              Defaults to 0.
+            * false_northing - Y offset from planar origin in metres.
+                               Defaults to 0.
+            * standard_parallels - The one or two latitudes of correct scale.
```

Comment:
Yes, it appears that they should be the same thing. I don't really have a
preference either way.
On Feb 16, 2015 9:00 AM, "Phil Elson" <notifications@github.com> wrote:

> In lib/cartopy/crs.py
> <https://github.com/SciTools/cartopy/pull/554#discussion_r24751856>:
>
> > +    the conterminous United States.
> > +    """
> > +
> > +    def __init__(self, central_longitude=0.0, central_latitude=0.0,
> > +                 false_easting=0.0, false_northing=0.0,
> > +                 standard_parallels=(20.0, 50.0), globe=None):
> > +        """
> > +        Kwargs:
> > +
> > +            * central_longitude - The central longitude. Defaults to 0.
> > +            * central_latitude - The central latitude. Defaults to 0.
> > +            * false_easting - X offset from planar origin in metres.
> > +                              Defaults to 0.
> > +            * false_northing - Y offset from planar origin in metres.
> > +                               Defaults to 0.
> > +            * standard_parallels - The one or two latitudes of correct scale.
>
> Unless I'm mistaken, this has also been called secant_latitudes in
> https://github.com/SciTools/cartopy/blob/master/lib/cartopy/crs.py#L1011.
> I'd like to have consistency in terminology - that either means changing
> this, or deprecating the other.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SciTools/cartopy/pull/554/files#r24751856>.
>